airflow/api/common/experimental/mark_tasks.py

```diff
@@ -335,18 +336,18 @@ def set_dag_run_state_to_failed(dag, execution_date, commit=False, session=None)
 
     # Mark the dag run to failed.
     if commit:
-        _set_dag_run_state(dag.dag_id, execution_date, State.FAILED, session)
+        _set_dag_run_state(dag.dag_id, execution_date, TaskInstanceState.FAILED, session)
 
-    # Mark only RUNNING task instances.
+    # Mark only running task instances.
     task_ids = [task.task_id for task in dag.tasks]
     tis = (
         session.query(TaskInstance)
         .filter(
             TaskInstance.dag_id == dag.dag_id,
             TaskInstance.execution_date == execution_date,
             TaskInstance.task_id.in_(task_ids),
+            TaskInstance.state.in_(State.running),
         )
-        .filter(TaskInstance.state == State.RUNNING)
     )
```

Comment:
(Note for other reviewers) From what I can tell, this is the only functional change; all others are Mypy fixes.

Reply:
Yes. That's right.